### ISSUE TITLE
Filling out fields incorrectly in Amcrest camera software

### ISSUE THREAD

**Issue by STaRDoGG** [bitwarden/clients#3382]
### Steps To Reproduce

(the usual)
go to the login page
click the Chrome browser extension
click the Login for the saved Amcrest form

### Expected Result

Fills in the login and pass as expected

### Actual Result

it fills the password field in with the saved user name (leaves the user field empty).

### Screenshots or Videos

I saved a screenshot of the form with it's form names in case it helps. Maybe there's a regex in BW's software that's missing this match?

![image](https://user-images.githubusercontent.com/1524526/186755664-aff4c677-41cc-49cb-87a6-fd8a79efc7a1.png)

### Additional Context

In this software, if the user name is previously filled in it'll sometimes autofill the username, but I think that probably shouldn't have any effect on this issue, since I think(?) BW normally fills in all fields saved anyway.

### Operating System

Windows, Linux

### Web Browser

Chrome

### Build Version

2020.8.0

**Comment by djsmith85** [comment]
Hi @STaRDoGG,

Thanks for reporting this issue. This is a known problem that affects a number of sites, and we’re working on improving this feature. To help us track and analyze affected sites, please lodge a report using the Google Form mentioned in this issue: #1621. Please also direct any discussion or questions to that issue. 

In the meantime, please have a look at our help-site for [autofill](https://bitwarden.com/help/auto-fill-custom-fields/) and specifically [linked custom fields}(https://bitwarden.com/help/auto-fill-custom-fields/#using-linked-custom-fields). This technique can be useful with sites that have trouble auto-filling like in your case you mentioned above.

This issue will now be closed.

**Comment by STaRDoGG** [comment]
@djsmith85 thanks, worked great. A couple things I noticed while doing this;

-  The browser extension was kinda wonky when trying to add the linked stuff (unless I'm doing it wrong and missed something) I could select the user/pass dropdown, but saw nowhere to enter the form fields to link to them. I ended up doing it on the server itself instead, which was a piece of cake.

- After testing the newly linked fields (that work now) I get the bar at the top asking if I want to save the new credentials to bw. As a test I went ahead and saved it as a new login, and tested it, but it just does the same as before, so in essence the "save bar" become a bit of a "popup annoyance" because I have to click close on it every time. For not not a big deal since the login works and you're aware of the issue, but thought worth mentioning.
